### DISCUSSION TITLE
Alchemy JSON to PDF Character Sheet

### DISCUSSION THREAD

**Discussion by jmitchellkelley** [thatbudakguy/ddb2alchemy#40] [General]
Would this be possible? Big Ask haha

**Comment by voidrender** [comment]
Yeah, this would be possible. I'm not sure how much work it is. We actually plan to build an export to PDF function into Alchemy so you can take your characters to the table, but we won't be getting to it this year. 

Meanwhile, I bet the community could come up with something!

**Comment by jmitchellkelley** [comment]
Hey Isaac!

Glad to hear direct to PDF would be a thing eventually! If you had to guess
early or late next year - could you let me know what your thoughts are? I
know it would be super useful to my group to have that functionality :)

**Reply by voidrender** [reply]
_Probably_ later next year, but tough to say. Right now we're laser-focused on getting v1 out on time! 😅
